api/swagger/swagger-v1.yaml

```diff
@@ -2045,6 +2045,106 @@ paths:
         "500":
           description: Server error
           content: {}
+
+  /users/{id}/coins:
+    get:
+      tags:
+      - users
+      description: 'Gets a list of the coins owned by the user and their balances'
+      parameters:
+      - name: id
+        in: path
+        description: A User ID
+        required: true
+        schema:
+          type: string
+      operationId: Get User Coins
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object 
+                properties:
+                  data:
+                    type: array
+                    items:
+                      type: object
```

Comment:
Just curious if this gets generated in SDK as just an `object`. I think you have to use a schema ref to get the generator to spit out a separate type for this field.

Reply:
https://github.com/AudiusProject/audius-protocol/pull/12579 seems ok?

Reply:
renamed anyway

Reply:
huh yeah I guess it just figures out a name for it then.
So I guess we keep schemas for types that are shared between endpoints?

Reply:
Oh I renamed them after to be consistent, the names it generated are kinda ugly